.travis.yml

```diff
@@ -9,63 +9,32 @@ addons:
+      if: type = pull_request
       script: npm run build:release
       addons: skip
     #Integration tests
-    - stage: integ_audio
-      if: (type = pull_request) OR (type = push AND branch != master)
-      before_install: script/needs-integration-test || travis_terminate 0
-      install:
-        - source script/travis-awscli-installation
-        - integration/js/script/install-kite
-      script: npm run test:integration-audio
-    - stage: integ_video
-      if: (type = pull_request) OR (type = push AND branch != master)
-      before_install: script/needs-integration-test || travis_terminate 0
-      install:
-        - source script/travis-awscli-installation
-        - integration/js/script/install-kite
-      script: npm run test:integration-video
-    - stage: integ_meeting_quit
-      if: (type = pull_request) OR (type = push AND branch != master)
-      before_install: script/needs-integration-test || travis_terminate 0
-      install:
-        - source script/travis-awscli-installation
-        - integration/js/script/install-kite
-      script: npm run test:integration-meeting-end
-    - stage: integ_screen_share
-      if: (type = pull_request) OR (type = push AND branch != master)
-      before_install: script/needs-integration-test || travis_terminate 0
-      install:
-        - source script/travis-awscli-installation
-        - integration/js/script/install-kite
-      script: npm run test:integration-screen-share
-    - stage: integ_content_share
-      if: (type = pull_request) OR (type = push AND branch != master)
-      before_install: script/needs-integration-test || travis_terminate 0
-      install:
-        - source script/travis-awscli-installation
-        - integration/js/script/install-kite
-      script: npm run test:integration-content-share
-    - stage: integ_data_message
-      if: (type = pull_request) OR (type = push AND branch != master)
-      before_install: script/needs-integration-test || travis_terminate 0
-      install:
-        - source script/travis-awscli-installation
-        - integration/js/script/install-kite
-      script: npm run test:integration-data-message
-    - stage: integ_meeting_readiness_checker
-      if: (type = pull_request) OR (type = push AND branch != master)
-      before_install: script/needs-integration-test || travis_terminate 0
-      install:
-        - source script/travis-awscli-installation
-        - integration/js/script/install-kite
-      script: npm run test:integration-meeting-readiness-checker
+    - stage: integ_group_1
+      - name: integ_audio
+        if: type = pull_request
+        before_install: script/needs-integration-test || travis_terminate 0
+        install:
+          - source script/travis-awscli-installation
+          - integration/js/script/install-kite
+        script: npm run test:integration-audio
+      - name: integ_content_share
+        if: type = pull_request
```

Comment:
dont need this anymore?

Reply:
Ignore that. I tried to remove code to test where I made Travis syntax error.

Reply:
since we are only going to run the travis build on pull requeusts, you can delete the 

`if: type = pull_request`

for each test